tests/Unit/Modifier/Category/ActiveModifierTest.php

```diff
@@ -33,7 +34,9 @@ public function testModifyTypeIfActive()
         $modelMock->method('getCoreTableName')
             ->willReturn('oxcategories_test');
 
-        $modifier = new ActiveModifier($databaseMock, $modelMock);
+        EshopRegistry::getUtilsObject()::setClassInstance(Category::class, $modelMock);
```

Comment:
Never call a static class in an instance.

If the instance of `EshopRegistry::getUtilsObject()` can change use
```suggestion
        $setClassInstance = [get_class(EshopRegistry::getUtilsObject()), 'setClassInstance'];
        $setClassInstance(Category::class, $modelMock);
```

Or if it is always the same:
```suggestion
        OxidEsales\Eshop\Core\UtilsObject::getUtilsObject(Category::class, $modelMock);
```